source/client/options_impl.h

```diff
@@ -178,6 +183,7 @@ class OptionsImpl : public Options, public Envoy::Logger::Loggable<Envoy::Logger
   std::string latency_response_header_name_;
   absl::optional<Envoy::SystemTime> scheduled_start_;
   absl::optional<std::string> execution_id_;
+  std::vector<envoy::config::core::v3::TypedExtensionConfig> user_defined_output_plugin_configs_{};
```

Comment:
Why do we need the curly braces?

Reply:
Got it. Good to know this defensive way to initialize private members. Sounds like this could apply to all types, not just non-primitive types?